src/lib.rs

```diff
@@ -18,6 +18,8 @@
 //! Calling the `Hasher::new` constructor at runtime will perform a feature detection to select the most
 //! optimal implementation for the current CPU feature set.
 
+#![cfg_attr(target_arch = "aarch64", feature(asm,aarch64_target_feature))]
```

Comment:
Given that enabling these features requires nightly, and this crate should generally work on stable by default, what do you think about exposing a `nightly` cargo flag to feature gate this?

Reply:
Yeah, sure